pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -0,0 +1,645 @@
+						v1beta1constants.LabelApp:                            DeploymentName,
+						"networking.gardener.cloud/to-shoot-networks":        v1beta1constants.LabelNetworkPolicyAllowed,
+						v1beta1constants.LabelNetworkPolicyToDNS:             v1beta1constants.LabelNetworkPolicyAllowed,
+						v1beta1constants.LabelNetworkPolicyToPrivateNetworks: v1beta1constants.LabelNetworkPolicyAllowed,
+						v1beta1constants.LabelNetworkPolicyFromPrometheus:    v1beta1constants.LabelNetworkPolicyAllowed,
+					},
+					Annotations: map[string]string{
+						"checksum/secret-" + tlsAuthSecret.Name: v.secrets.TLSAuth.Checksum,
+						"checksum/secret-" + dhSecret.Name:      v.secrets.DH.Checksum,
+					},
+				},
+				Spec: corev1.PodSpec{
+					PriorityClassName: "system-cluster-critical",
+					DNSPolicy:         corev1.DNSDefault, // make sure to not use the coredns for DNS resolution.
+					Containers: []corev1.Container{
+						{
+							Name:            DeploymentName,
+							Image:           v.imageVPNSeedServer,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							Ports: []corev1.ContainerPort{
+								{
+									Name:          "tcp-tunnel",
+									ContainerPort: 1194,
+									Protocol:      corev1.ProtocolTCP,
+								},
+							},
+							SecurityContext: &corev1.SecurityContext{
+								Capabilities: &corev1.Capabilities{
+									Add: []corev1.Capability{
+										"NET_ADMIN",
+									},
+								},
+								Privileged: pointer.BoolPtr(true),
+							},
+							Env: []corev1.EnvVar{
+								{
+									Name:  "SERVICE_NETWORK",
+									Value: v.serviceNetwork,
+								},
+								{
+									Name:  "POD_NETWORK",
+									Value: v.podNetwork,
+								},
+							},
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("100m"),
+									corev1.ResourceMemory: resource.MustParse("100Mi"),
+								},
+								Limits: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("1"),
+									corev1.ResourceMemory: resource.MustParse("1Gi"),
+								},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      DeploymentName,
+									MountPath: MountPathVpnSeedServer,
+								},
+								{
+									Name:      VpnSeedServerTLSAuth,
+									MountPath: MountPathTLSAuth,
+								},
+								{
+									Name:      VpnSeedServerDH,
+									MountPath: MountPathDH,
+								},
+							},
+						},
+						{
+							Name:            "envoy-proxy",
+							Image:           v.imageAPIServerProxy,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							SecurityContext: &corev1.SecurityContext{
+								Capabilities: &corev1.Capabilities{
+									Add: []corev1.Capability{
+										"NET_BIND_SERVICE",
+									},
+								},
+							},
+							Command: []string{
+								"envoy",
+								"--concurrency",
+								"2",
+								"-c",
+								"/etc/envoy/envoy.yaml",
```

Comment:
Define a constant for `/etc/envoy` and use it here + below in your mount path

Reply:
Also, define a constant for `envoy.yaml` and use it here and below

Reply:
Done.